platform/linux/src/headless_display_egl.cpp

```diff
@@ -31,8 +31,23 @@ HeadlessDisplay::Impl::Impl() {
         throw std::runtime_error("eglBindAPI() failed");
     }
 
+#if !defined(__ANDROID__)
     // This shouldn't matter as we're rendering to a framebuffer.
     const EGLint attribs[] = { EGL_NONE };
+#else
+    const EGLint attribs[] = {
```

Comment:
From my local test runs, it seems that the EGL implementation used by the Android emulator does not support FBOs, unfortunately.